site/src/components/pages/user/tabs.tsx

```diff
@@ -49,6 +49,7 @@ export const TabHeader: FunctionComponent<TabHeaderProps> = ({
 
         return (
           <BaseLink
+            data-testid={`profile-page-${value}-tab`}
```

Comment:
[optional] Might be worth keeping `data-testid` values and indivisible symbols as this can ease their search. We used this principle in my previous company for `data-testid`s and i18n keys. We can introduce a lookup object where value maps to data-testid.

Alternatively, we can stick with just one `data-testid` and select by `data-testid` + text

Reply:
Added in [33301ee](https://github.com/blockprotocol/blockprotocol/pull/456/commits/33301ee41c157457768abd4ea917cb38f5225dab)